pkg/conversion/scheme.go

```diff
@@ -375,6 +380,14 @@ func (s *Scheme) Converter() *Converter {
 	return s.converter
 }
 
+// WithConversions returns a scheme with additional conversion functions
```

Comment:
Replace, although in the specific case there is no overlap (runtime.Object
is defined as a conversion only for one path)


On Dec 1, 2015, at 7:24 PM, Daniel Smith <notifications@github.com> wrote:

In pkg/conversion/scheme.go
<https://github.com/kubernetes/kubernetes/pull/17922#discussion_r46361588>:

> @@ -375,6 +380,14 @@ func (s *Scheme) Converter() *Converter {
>  	return s.converter
>  }
>
> +// WithConversions returns a scheme with additional conversion functions

Do the functions replace or get appended to the existing ones?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/17922/files#r46361588>.